services/export/shared/helpers/chain.js

```diff
@@ -35,8 +36,20 @@ const resolveReceivingChainID = (tx, currentChainID) => tx
 	? tx.params.receivingChainID
 	: currentChainID;
 
+const getUniqueChainIDs = async (txs) => {
+	const chainIDs = [];
+
+	txs.forEach(tx => {
+		if (tx.sendingChainID) chainIDs.push(tx.sendingChainID);
+		if (tx.receivingChainID) chainIDs.push(tx.receivingChainID);
+	});
+
+	return Array.from(new Set(chainIDs));
+};
```

Comment:
Can be optimised by initialising chainIDs as a Set.

```suggestion
const getUniqueChainIDs = async (txs) => {
	const chainIDs = new Set();

	txs.forEach(tx => {
		if (tx.sendingChainID) chainIDs.add(tx.sendingChainID);
		if (tx.receivingChainID) chainIDs.add(tx.receivingChainID);
	});

	return Array.from(chainIDs);
};
```